plugins/catalog-backend/src/service/request/parseQueryEntitiesParams.test.ts

```diff
@@ -130,18 +123,14 @@ describe('parseQueryEntitiesParams', () => {
       const parsedObj = parseQueryEntitiesParams(
         {},
       ) as QueryEntitiesCursorRequest;
-      expect(parsedObj.limit).toBeUndefined();
       expect(parsedObj.fields).toBeUndefined();
     });
 
-    it.each([
-      {
-        limit: 'asd',
+    it.each([{ cursor: [] }, { fields: [4] }])(
+      'should throw if some parameter is not valid %p',
+      params => {
+        expect(() => parseQueryEntitiesParams(params)).toThrow();
       },
-      { cursor: [] },
-      { fields: [4] },
-    ])('should throw if some parameter is not valid %p', params => {
-      expect(() => parseQueryEntitiesParams(params)).toThrow();
-    });
+    );
```

Comment:
Yup, I wonder too if it would be worth using something like https://www.npmjs.com/package/jest-openapi to test these cases, especially for errors as those are probably where the spec is missing the most.

Reply:
Do you want to have `jest-openapi` as part of this PR? I think it would make sense to separate it out to a second PR that's more focused. I can add a test case for the loss of validation that happens here if needed.

Reply:
I'll add the missing test case to the router, just to ensure we have the same coverage.